web/packages/teleport/src/Sessions/SessionList/SessionList.tsx

```diff
@@ -112,7 +112,7 @@ const renderJoinCell = ({
   showActiveSessionsCTA,
 }: renderJoinCellProps) => {
   const { joinable } = kinds[kind];
-  if (!joinable || participantModes.length === 0) {
```

Comment:
Is this second check handled on the backend now?

Reply:
No, previously we would completely hide the join button if the user had access to no participant modes. However since we added a better join menu with descriptions as well as a "You do not have permission to join as a ____" disclaimer for modes you don't have access to, we can now show the button and just have all the options greyed out. This is less confusing for a user who might be wondering why the join button isn't there. 

This change should've actually been made in https://github.com/gravitational/teleport/pull/24961 but I had forgotten to and only noticed it here while testing.